src/pages/overview/pdf-services-api/howtos/ocr-pdf.md

```diff
@@ -212,42 +216,45 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
 // mvn -f pom.xml exec:java Dexec.mainClass=com.adobe.pdfservices.operation.samples.ocrpdf.OcrPDFWithOptions
  
   public class OcrPDFWithOptions {
- 
     // Initialize the logger.
     private static final Logger LOGGER = LoggerFactory.getLogger(OcrPDFWithOptions.class);
  
     public static void main(String[] args) {
  
         try {
- 
-            // Initial setup, create credentials instance.
-            Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-                .withClientId("PDF_SERVICES_CLIENT_ID")
-                .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                .build();
- 
-            //Create an ExecutionContext using credentials and create a new operation instance.
-            ExecutionContext executionContext = ExecutionContext.create(credentials);
-            OCROperation ocrOperation = OCROperation.createNew();
- 
-            // Set operation input from a source file.
-            FileRef source = FileRef.createFromLocalFile("src/main/resources/ocrInput.pdf");
-            ocrOperation.setInput(source);
- 
-            // Build OCR options from supported locales and OCR-types and set them into the operation
-            OCROptions ocrOptions = OCROptions.ocrOptionsBuilder()
+            // Initial setup, create credentials instance
+            Credentials credentials = new ServicePrincipalCredentials(
+                    System.getenv("PDF_SERVICES_CLIENT_ID"),
+                    System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+
+            // Creates a PDF Services instance
+            PDFServices pdfServices = new PDFServices(credentials);
+
+            // Creates an asset from source file and upload
```

Comment:
creates.  ---- > uploads.  ( create/creates get/gets submit/submits upload/uploads

Reply:
As discussed, IG this is from already reviewed samples.